app/scripts/controllers/pod.js

```diff
@@ -191,6 +193,12 @@ angular.module('openshiftConsole')
       return terminals;
     };
 
+    var updateContainersYet = function(pod) {
+      if ($scope.containersRunning(pod.status.containerStatuses) > 0 ) {
+        $scope.noContainersYet = false;
```

Comment:
Instead of an if check that sets a boolean, just make this a one liner

```
$scope.noContainersYet = $scope.containersRunning(pod.status.containerStatuses) === 0; 
```